core/server/apps/subscribers/lib/router.js

```diff
@@ -44,9 +45,13 @@ function honeyPot(req, res, next) {
     next();
 }
 
+function validateUrl(url) {
+    return validator.isEmptyOrURL(url) ? url : '';
+}
+
 function handleSource(req, res, next) {
-    req.body.subscribed_url = req.body.location;
-    req.body.subscribed_referrer = req.body.referrer;
+    req.body.subscribed_url = validateUrl(req.body.location);
```

Comment:
we already have a validator on the schema for subscribed_url. can you explain we why need that line here?